nested.py

```diff
@@ -3,20 +3,42 @@
 """
 Module docstring: One line description of what your program does.
 """
-__author__ = "???"
+__author__ = "Janelle Kuhns"
 
 import sys
 
 
 def is_nested(line):
+    stack = []
```

Comment:
So, you are right on the money with the idea of a stack = []
However, we can check for brackets like this: ['(', '[', '{' ...]
and then make a list of closing brackets as well (Or an objects, up to you) This way we don't have to do if, else if, else if ...

Keep in mind that '(*' is two characters, but needs to be counted as one. Perhaps we might want to consider some way of counting brackets like they are some sort of 'token' where each token is one index of the string.